src/main/java/core/basesyntax/handler/impl/ReturnTransactionHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.handler.impl;
+
+import core.basesyntax.handler.TransactionHandler;
+import java.util.Map;
+
+public class ReturnTransactionHandler implements TransactionHandler {
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the redundant empty line at the start of the class definition.